packages/suite/src/storage/migrations/index.ts

```diff
@@ -190,7 +190,7 @@ export const migrate: OnUpgradeFunc<SuiteDBSchema> = async (
     if (oldVersion < 21) {
         // do the same thing as in blockchain-link's transformTransaction
         let cursor = await transaction.objectStore('txs').openCursor();
-        const symbolsToExclude = ['eth', 'etc', 'xrp', 'trop', 'txrp'];
+        const symbolsToExclude = ['eth', 'etc', 'xrp', 'txrp'];
```

Comment:
this does not feel right. changing old migration. here, probably nothing would happen, but this practice could end up in state where subsequent migration would rely on something changed by an older migration

Reply:
done squashed